docs/data-sources/prefix.md

```diff
@@ -17,11 +17,14 @@ description: |-
 
 ### Optional
 
-- `cidr` (String)
-- `vrf_id` (Number)
+- `cidr` (String) At least one of `cidr`, `vid`, `vrf_id` or `vlan_id` must be given.
```

Comment:
Hm. I see that the existing data source uses the `cidr` attribute name for the.. well... cidr of a prefix, but netbox itself (and the `netbox_prefix` **resources**) actually use the name `prefix` for this.

I suggest we stick with the netbox standard and use `prefix` and deprecate the `cidr` attribute.

This comment applies to all instances of `cidr` in this PR.

Reply:
i had this doubt too,i agree with your approach